src/ui/src/js/configs/routes.js

```diff
@@ -62,16 +61,16 @@ export default function routeConfig(
     })
     .state('base.about', {
       url: 'about',
-      templateUrl: basePath + 'about.html',
+      templateUrl: 'about.html',
```

Comment:
Just to make sure that I am tracking this properly, the removal of `basePath` is possible because of the change in the webpack? As long as this works when we publish the package I am good.

Reply:
Yeah, that's correct. I didn't really take the time to understand that part of the webpack config when I created it, so it was kind of screwy and wouldn't work with child directories.